### DISCUSSION TITLE
different booking form

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#158] [Q&A]
Hi Franky
I would like the booking form to be different template depending on the category of the event. It seems that the shortcode or the placeholder has no parameter. Is this possible and how can I do it?
Have a nice day,
Frédéric

**Comment by liedekef** [comment]
The booking form template is set per event, not per category (which is another property of an event).
if you want to show different things based on event properties, use [eme_if] and check the value of #_EVENTCATEGORIES or alike using the conditionial check "incsv"

**Comment by f14m07** [comment]
Can I use [eme_if] in the 'Default single event format' ?

**Comment by liedekef** [comment]
yes you can, but in your case you'll want to use it in the booking template of course.

**Comment by f14m07** [comment]
Thanks, it works fine. In the same way, is it possible to prevent the sending of e-mails for a category of event?

**Comment by liedekef** [comment]
in the email templates: if the subject of body is empty, the email will not be sent. So if you create your logic that way, it will work too.
Although using templates, it seems much easier to me to assign different templates to events, but that's all up to you.

**Comment by f14m07** [comment]
Initially I had only one category of events. Now I am looking to implement another virtual event category. The event will become in the first category if an organizer accepts to take this event under his responsibility. One organizer will only do this if enough people sign up for the virtual event.

**Comment by f14m07** [comment]
I would like to avoid doing too much manual manipulation in the backend
